optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/PureSingleStatistic.java

```diff
@@ -30,7 +30,7 @@
 import org.optaplanner.benchmark.impl.statistic.single.pickedmovetypestepscore.PickedMoveTypeStepScoreDiffSingleStatistic;
 
 /**
- * 1 statistic of {@link SingleBenchmarkResult}
+ * 1 statistic of {@link SingleBenchmarkResult}.
```

Comment:
I've just seen a javadocs guidelines page that also uses dots there in an example, so looks like you're right :)
  http://www.oracle.com/technetwork/articles/java/index-137868.html
My memory says the opposite though - we might want to consult Gemma about this - product docs probably has a guideline about this sort of thing.